en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+    !!! Tip
+        If you want to filter the report content by the API consumer instead of the application, click on the toggle switch to the right of the **Application** field, and then select the required consumer. Only consumers who have subscribed to one or more of the selected APIs are displayed in the list. You can either select one, or all of them.
+        
+3. Select the required year and the month in the **Year** and **Month** fields to specify the time interval for which you want to generate the report.
+
+4. Click **Download**, and select the required format.
+
+The report is downloaded in the specified format. The following is a sample report generated in PDF format.
+
+![Sample report](../assets/img/insights/custom-report-sample.png){.cInlineImage-full}
+
+The contents of the report are as follows:
+
+| **Item** | **Description** |
+|-------------------------|--------------------------------------------------------------------------------------------|
+| **Generated Time**      | The date and the time the report was generated.                                            |
+| **Organization**        | The organization to which the report applies. Each report always applies to a specific organization. |
+| **Environment**         | The environment to which the report applies. Each report always applies to a specific environment.  |
+| **Tenant**              | The tenant to which the report applies. Each report always applies to a specific tenant.            |
+| **Reporting Period**    | The time interval for which the report is generated.                                       |
+| **Total request count** | The total request count received by all selected APIs during the report time interval.     |
+
+The table in the report contains the following rows:
+
+| **Item** | **Description** |
+|-----------------------|------------------------------------------------------------------------------|
+| **API Name**          | The name of the API.                                                         |
+| **Application name:** | The name of the application. The name of the subscriber is also provided within brackets. |
+| **Usage Count**       | The request count.                                                           |
+
+#### Download pre-generated reports
+
+These widgets allow you to download monthly reports for the last three months.
+
+![Pre-generated reports](../assets/img/insights/configured-report.png){.cInlineImage-full}
+
+To download a pre-generated report, click **Download** for the relevant month and then click on the required format.
+
+### Alert Configuration
+
+This page allows you to configure alerts. These alerts allow you to proactively monitor your API ecosystem and take corrective measures for any abnormalities you find.
```

Comment:
We use the word "section" to refer to a page in documentation. "Page" has always been used to refer to a page in the UI.

Reply:
Yes, but it's not pretty straight forward, and we would know. But, for someone who is not a TW, would not know the usage of the terms right? better to avoid ambiguity?

Reply:
Referring to this kind of UI element as "page" is the practice we normally follow and we have already used it consistently in Choreo documentation in many places. See "**Integrations** page" in [1] and "**Services** page" in [2].

If we change it here, we would need to change it everywhere.

[1] https://wso2.com/choreo/docs/integrations/build-your-first-integration/
[2] https://wso2.com/choreo/docs/services/create-your-first-service/

Reply:
Maybe giving the name of the page before the word "page" instead of just saying "This page" would help? We will;l be repeating the title though.